run_simulations.py

```diff
@@ -1,16 +1,47 @@
 from scrabbler.simulation import Simulation
 from scrabbler.player import Player
 from scrabbler.strategy import BaselineStrategy
+from scrabbler.ABStrategy import ABStrategy
+import matplotlib.pyplot as plt
 import sys
 
 def main():
-    n = 1
+    n = 10
+    p0scores = []
+    p1scores = []
+
     if len(sys.argv) > 1 and sys.argv[1].startswith("--n="):
         n = int(sys.argv[1][len("--n="):])
-    for i in range(n):
-        player1 = Player(BaselineStrategy())
-        player2 = Player(BaselineStrategy())
-        Simulation.simulate_game(player1, player2)
+    for i in range(n ):
+        player1 = Player(midgame_strategy=BaselineStrategy(), endgame_strategy=ABStrategy(), name="AB Pruning Endgame")
```

Comment:
This is okay for now, but we may want to add command line arguments to this script that allow us to pass in the strategies used at runtime, instead of hardcoding a baseline strategy vs. an AB pruning strategy in the code.